packages/react/src/Token/_RemoveTokenButton.module.css

```diff
@@ -0,0 +1,52 @@
+.TokenButton {
+  display: inline-flex;
+  padding: 0;
+  margin-left: var(--base-size-4);
+  font-family: inherit;
+  color: currentColor;
+  text-decoration: none;
+  cursor: pointer;
+  user-select: none;
+  background-color: transparent;
+  border: 0;
+  border-radius: var(--borderRadius-full);
+  justify-content: center;
+  align-items: center;
+  appearance: none;
+  align-self: baseline;
+}
+
+.TokenButton[data-size='small'] {
+  width: var(--base-size-16);
+  height: var(--base-size-16);
+}
+
+.TokenButton[data-size='medium'] {
+  width: 20px;
+  height: 20px;
+}
+
+.TokenButton[data-size='large'] {
+  width: var(--base-size-24);
+  height: var(--base-size-24);
+}
+
+.TokenButton[data-size='xlarge'] {
+  width: var(--base-size-32);
+  height: var(--base-size-32);
+}
+
+.Bigger {
+  margin-left: var(--base-size-8);
```

Comment:
Curious could you add this to the `.TokenButton[data-size='xlarge']` and `.TokenButton[data-size='large']` selectors instead of adding `.Bigger`?

Reply:
Good callout! [Update](https://github.com/primer/react/pull/5251/commits/3d65c721f0d6f1871a30feae507872ce88c17e56)